Runnect-iOS/Runnect-iOS/Presentation/MyPage/VC/InfoVC/ActivityRecordInfoTableView/ActivityRecordInfoTVC.swift

```diff
@@ -121,6 +121,14 @@ extension ActivityRecordInfoTVC {
         activityRecordMapImage.setImage(with: model.image)
     }
     
+    func updateSelectionCellUI(isSelected: Bool) {
+        if isSelected {
+            activityRecordContainerView.image = ImageLiterals.imgRecordContainerSelected
+        } else {
+            activityRecordContainerView.image = ImageLiterals.imgRecordContainer
+        }
```

Comment:
```suggestion
        activityRecordContainerView.image = isSelected ? 이미지1 : 이미지2
```
이렇게 삼항 연산자로도 쓸 수 있을 거 같아요!

Reply:
감사합니다! 반영했어요!